application/src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -83,10 +81,14 @@ public class TimetableSnapshot {
 
   /**
    * During the construction phase of the TimetableSnapshot, before it is considered immutable and
-   * used in routing, this Set holds all timetables that have been modified and are waiting to be
-   * indexed. This field will be set to null when the TimetableSnapshot becomes read-only.
+   * used in routing, this Map holds all timetables that have been modified and are waiting to be
+   * indexed.
+   * A real-time timetable overrides the scheduled timetable of a TripPattern for only a single
+   * service date. There can be only one overriding timetable per TripPattern and per service date.
+   * This is enforced by indexing the map with a pair (TripPattern, service date).
+   * This field will be set to null when the TimetableSnapshot becomes read-only.
```

Comment:
I don't think I get it. What is being set to null? I don't see it.

Reply:
The last sentence was copied from the original Javadoc, and you're right, this was not correct: the collection is cleared, not set to null.
Javadoc fixed.